### PR TITLE
docs(components): Update Text Proposal

### PR DESCRIPTION
**Pull request by eddysims**
## Motivations

We need to make some modifications to the `Text` component. This is a proposal for how to most effectively make those changes.

### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/proposals/Text/Text_Mods.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+In this case the new props could look like this:
+
+| prop      | type   | options                                       |
+| --------- | ------ | --------------------------------------------- |
+| size      | string | "small", "base", "large"                      |
+| variation | string | "success", "error", "subdued", "warn", "info" |
+
+## Opinion
+
+In my opinion, the `Multiple Props` is the way to go. I think that the breaking
```
Comment (JeffMarvin) [inline docs/proposals/Text/Text_Mods.md:49]: @eddysims Agreed. I wonder if we should choose something other than variation and then just deprecate that prop? We could map the existing variations over to use the new props....

Comment (eddysims) [reply docs/proposals/Text/Text_Mods.md:49]: @JeffMarvin I actually dont think that variation is that bad. We currently use it in Button and Toast. thoughts?

Comment (JeffMarvin) [reply docs/proposals/Text/Text_Mods.md:49]: Yeah... I just went through and looked at those Atlantis components, and I think `variation` is what we want. 
What's the scope of the breaking change?

File: docs/proposals/Text/Text_Mods.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+styling with the `error` styles and then over riding them with the `info`
+styles.
+
+### Multiple Props
+
+Text could take 2 props instead of one. `size` and `variation`. This would allow
+for us to do things like `<Text size="small" variation="info">`.
+
+My concern with this is that it may result as a breaking change. The reason
+being is that the `intro` variation would need to be moved into the `size` prop.
```
Comment (seanhealy) [inline docs/proposals/Text/Text_Mods.md:38]: We could keep `default` and `intro` in as variations and deprecate them for now. Then we can clean them up with a breaking change later perhaps?

**Pull request merged**